ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/required-permission/deprovision/cucushift-installer-rehearse-aws-ipi-required-permission-deprovision-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-required-permission-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+  - ref: aws-deprovision-cco-manual-users-static
```

Comment:
@jianlinliu how about renaming this step to a more generic name, like `aws-deprovision-users-and-policies`

Reply:
yeah, agree.